<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteAsWindow</class>
 <widget class="QMainWindow" name="DeleteAsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Eliminar Aseguradora</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="input_id">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>382</width>
      <height>39</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
          border: 2px solid #5dade2;
          border-radius: 10px;
          padding: 8px;
          font-size: 14px;
          background-color: white;
      }
      QPushButton {
          
	background-color: rgb(170, 0, 0);
          color: white;
          font-size: 14px;
          font-weight: bold;
          padding: 10px 20px;
          border-radius: 10px;
      }
      QPushButton:hover {
          
	background-color: rgb(227, 28, 10);
      }</string>
    </property>
    <property name="placeholderText">
     <string>ID de la compa√±ia</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_eliminar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>255</y>
      <width>382</width>
      <height>37</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
          border: 2px solid #5dade2;
          border-radius: 10px;
          padding: 8px;
          font-size: 14px;
          background-color: white;
      }
      QPushButton {
          
	background-color: rgb(170, 0, 0);
          color: white;
          font-size: 14px;
          font-weight: bold;
          padding: 10px 20px;
          border-radius: 10px;
      }
      QPushButton:hover {
          
	background-color: rgb(227, 28, 10);
      }</string>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>42</y>
      <width>382</width>
      <height>162</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>e.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>567</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
